src/modules/checkIsValidUserInput.js

```diff
@@ -9,7 +9,19 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  switch (true) {
+    case userInput[0] === '0':
+      return false;
+
+    case isNaN(+userInput):
```

Comment:
The check `isNaN(+userInput)` is intended to ensure the input is numeric. However, this will not catch cases where the input is a numeric string with non-digit characters (e.g., '12a4'). Consider using a regular expression to ensure the input is strictly a 4-digit number.